src/constants/index.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+    DEFAULT_IMAGE: {
+        AVATAR: 'https://res.cloudinary.com/dq4xoyzib/image/upload/v1668956700/lgwweg5qvz3wydozqhkl.png',
+        BACKGROUND: 'https://res.cloudinary.com/dq4xoyzib/image/upload/v1668956700/lgwweg5qvz3wydozqhkl.png',
```

Comment:
Please find another image to be default avatar of the group, not copy from user's default avatar :)

Reply:
Already updated. Thank you @dlxkhang